src/webvr-polyfill.js

```diff
@@ -95,6 +95,18 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
 
   // Provide the VRDisplay object.
   window.VRDisplay = VRDisplay;
+
+  // Provide vrEnabled
+  var that = this;
+  Object.defineProperty(navigator, 'vrEnabled', {
```

Comment:
I thought about that. I suppose it makes sense if we can come up with a reliable way to detect it. Maybe request VR displays and return `false` if there's an error? What do you think?

Note that this needs to detect every time, since the attribute on the iframe could change at any time.